### PR TITLE
Dealing with 98

### LINKED ISSUE #98: Zoom to the content of IFC file not working
When loading point cloud data, the view moves to the loaded content immediately. This used to work on IFC data as well, but is not anymore.

### PR DESCRIPTION
**Pull request by pjanck**
Fixes #98 . 

The IFC model now provides a bounding box, so that the viewport can react appropriately. The geometry is also moved to (0,0,0), similar to the point cloud handling.

Bounding box should be a -must-have- attribute of any model, if we are to extend to multimodel support.

### CODE REVIEW

**Comment by kyq** [comment]
Working well. IFC models are properly positioned and shown.

Fixes #98.

File: Core/src/IfcGeometryConverter/ConverterBuw.h [hunk 1]
```diff
@@ -43,22 +43,63 @@ namespace OpenInfraPlatform
...
+				std::string toString() const {
+					return "min: (" + std::to_string(min_.x()) + ", " + std::to_string(min_.y()) + ", " + std::to_string(min_.z())
+					   + ") max: (" + std::to_string(max_.x()) + ", " + std::to_string(max_.y()) + ", " + std::to_string(max_.z()) + ")";
+				}
+				buw::Vector3d center() const { return 0.5 * (min_ + max_); }
+				buw::Vector3d min() const { return min_; }
+				buw::Vector3d& min() { return min_; }
+				buw::Vector3d max() const { return max_; }
+				buw::Vector3d& max() { return max_; }
+			};
```
Comment (hechth) [inline Core/src/IfcGeometryConverter/ConverterBuw.h]: This belongs into an own file or own class - best as a extension maybe to the carve bounding box or by having an own interface which has an implementation that uses carve

so oip::IBoundingBox -> BoundingBoxImpl which uses carve bounding box and forwards functionality.

**Review by hechth — CHANGES_REQUESTED** [review]
Please rework the bounding box to incorporate the functionality from carve or have a custom interface that has an implementation which forwards functionality.

File: Core/src/IfcGeometryConverter/ConverterBuw.h (hunk 1 above)
Comment (pjanck) [reply Core/src/IfcGeometryConverter/ConverterBuw.h]: done.

**Comment by kyq** [comment]
Works fine with both clouds and ifc models

**Pull request merged**